provisioning/roles/geerlingguy.drupal/tasks/build-composer-project.yml

```diff
@@ -1,4 +1,20 @@
 ---
+- name: Delete old /tmp/drupal-project
+  file:
+    path: "/tmp/drupal-project"
+    state: absent
+  when: not drupal_site_exists
```

Comment:
If we want to avoid being destructive at all cost we would need an additional `fail` task that checks if `drupal_site_exists` is false and `drupal_composer_install_dir` is not empty. Personally I feel `cp` is safe enough as eg `.git` directory wont be removed, nor any new files.